docs/side_quests/splitting_and_grouping.md

```diff
@@ -0,0 +1,1298 @@
+                            [
+                                grouping_key.subMap('id', 'interval'),
+                                normal,
+                                tumor
+                            ]
+                          }
+                          .view()
+    ```
+
+=== "Before"
+
+    ```groovy title="main.nf" linenums="25"
+    ch_combined_samples = ch_joined_samples.combine(ch_intervals)
+                        .map { grouping_key, normal, tumor, interval ->
+                            [
+                                grouping_key + [interval: interval],
+                                normal,
+                                tumor
+                            ]
+
+                        }
+                        .view()
+    ```
+
+Let's run it again and check the channel contents:
+
+```bash title="View grouped samples"
+nextflow run main.nf
+```
+
+```console title="View grouped samples"
+ N E X T F L O W   ~  version 24.10.5
+
+Launching `main.nf` [loving_escher] DSL2 - revision: 3adccba898
+
+[[id:sampleA, interval:chr1], [type:normal, bam:sampleA_rep1_normal.bam], [type:tumor, bam:sampleA_rep1_tumor.bam]]
+[[id:sampleA, interval:chr2], [type:normal, bam:sampleA_rep1_normal.bam], [type:tumor, bam:sampleA_rep1_tumor.bam]]
+[[id:sampleA, interval:chr3], [type:normal, bam:sampleA_rep1_normal.bam], [type:tumor, bam:sampleA_rep1_tumor.bam]]
+[[id:sampleA, interval:chr1], [type:normal, bam:sampleB_rep1_normal.bam], [type:tumor, bam:sampleB_rep1_tumor.bam]]
+[[id:sampleA, interval:chr2], [type:normal, bam:sampleB_rep1_normal.bam], [type:tumor, bam:sampleB_rep1_tumor.bam]]
+[[id:sampleA, interval:chr3], [type:normal, bam:sampleB_rep1_normal.bam], [type:tumor, bam:sampleB_rep1_tumor.bam]]
+[[id:sampleB, interval:chr1], [type:normal, bam:sampleC_rep1_normal.bam], [type:tumor, bam:sampleC_rep1_tumor.bam]]
+[[id:sampleB, interval:chr2], [type:normal, bam:sampleC_rep1_normal.bam], [type:tumor, bam:sampleC_rep1_tumor.bam]]
+[[id:sampleB, interval:chr3], [type:normal, bam:sampleC_rep1_normal.bam], [type:tumor, bam:sampleC_rep1_tumor.bam]]
+[[id:sampleC, interval:chr1], [type:normal, bam:sampleD_rep1_normal.bam], [type:tumor, bam:sampleD_rep1_tumor.bam]]
+[[id:sampleC, interval:chr2], [type:normal, bam:sampleD_rep1_normal.bam], [type:tumor, bam:sampleD_rep1_tumor.bam]]
+[[id:sampleC, interval:chr3], [type:normal, bam:sampleD_rep1_normal.bam], [type:tumor, bam:sampleD_rep1_tumor.bam]]
+```
+
+We can see that we have successfully isolated the `id` and `interval` fields, but not grouped the samples yet.
+
+!!! note
+
+    We are discarding the `replicate` field here. This is because we don't need it for further downstream processing. After completing this tutorial, see if you can include it without affecting the later grouping!
+
+Let's now group the samples by this new grouping element, using the [`groupTuple` operator](https://www.nextflow.io/docs/latest/operator.html#grouptuple).
+
+=== "After"
```

Comment:
I am not sure why, but this blocks does not render correctly (only After, before is fine)
<img width="680" alt="Screenshot 2025-04-30 at 12 06 13" src="https://github.com/user-attachments/assets/5b5f0fe6-8af7-4cc5-b67a-8c329226e05a" />